### PR TITLE
Complete Issue 2

### PR DESCRIPTION
**Pull request by achenqi**

### CODE REVIEW

**Comment by achenqi** [comment]
[Issue 2: Save Image Link](https://trello.com/c/5Fw9Eoc7/422-issue-2-save-image-link)

File: config/initializers/simple_form.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+
```
Comment (infiniteluke) [inline config/initializers/simple_form.rb:2]: It looks like the `rubocop -a` ran on this commit but not on the one that adds simple form. We should edit that commit to be formatted correctly so it doesn't show here.

File: app/controllers/images_controller.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class ImagesController < ApplicationController
+  def new
+    @image = Image.new
+  end
+
+  def create
+    @image = Image.new(image_params)
+    if @image.save
+      redirect_to @image.url
```
Comment (infiniteluke) [inline app/controllers/images_controller.rb:9]: this can be `redirect_to @image`

Comment (infiniteluke) [reply app/controllers/images_controller.rb:9]: Does this page exist? I don't see a `show.html.erb` or a `show` action in this controller

**Review by infiniteluke — CHANGES_REQUESTED** [review]

File: app/controllers/images_controller.rb (hunk 2 above)
Comment (infiniteluke) [reply app/controllers/images_controller.rb]: this is redirecting to the acutal img which is fine for now

**Review by infiniteluke — APPROVED** [review]

**Pull request merged**